docs/ops.html

```diff
@@ -147,28 +147,31 @@
 
 <h4><a id="basic_ops_consumer_group" href="#basic_ops_consumer_group">Managing Consumer Groups</a></h4>
 
-With the ConsumerGroupCommand tool, we can list, delete, or describe consumer groups. For example, to list all consumer groups across all topics:
+With the ConsumerGroupCommand tool, we can list, describe, or delete consumer groups. Note that deletion is only available when the group metadata is stored in
+ZooKeeper. There's no need to delete it when using the <a href="http://kafka.apache.org/documentation.html#newconsumerapi">new consumer API</a> (where
```

Comment:
Yeah, I knew that was too simplistic, I just copied it from the `ConsumerGroupCommand` tool description. Clarified it as per your suggestion in both places.